litestar/testing/helpers.py

```diff
@@ -342,7 +342,7 @@ def create_async_test_client(
     opt: Mapping[str, Any] | None = None,
     parameters: ParametersMap | None = None,
     plugins: OptionalSequence[PluginProtocol] = None,
-    preferred_validation_backend: Literal["pydantic", "attrs"] = "attrs",
+    preferred_validation_backend: Literal["pydantic", "attrs"] | None = None,
```

Comment:
Why are you modifying the test helpers and only them?

Reply:
Given that we recently reverted to using the pydantic signature models by default, it seemed strange to me that these would default to the attrs model. So my intention here was for the default to fall back to the application default, unless it is explicitly set to use one or the other.

Reply:
I see, but the application does not expect None. 

Can you switch these to `pydantic` in both places?

Reply:
These signatures are exactly the same as the app:

https://github.com/litestar-org/litestar/blob/4f734ea5970d414d49b92cf3c49988aa158cad30/litestar/app.py#L208

Happy to change, but my intention was to just mirror exactly what the app accepts. So should we change the app signaturej to be `preferred_validation_backend: Literal["pydantic", "attrs"] = "pydantic"`. At the moment the default is being set at the time that the value is passed into the config object, here:

https://github.com/litestar-org/litestar/blob/4f734ea5970d414d49b92cf3c49988aa158cad30/litestar/app.py#L352

Reply:
Lol, ok 😸